pkg/kubectl/scale.go

```diff
@@ -61,10 +55,17 @@ func ScalerFor(kind schema.GroupKind, jobsClient batchclient.JobsGetter, scalesG
 	case batch.Kind("Job"):
 		return &jobScaler{jobsClient} // Either kind of job can be scaled with Batch interface.
```

Comment:
@p0lyn0mial are you up for removing this in a follow-up pull.  @soltysh got it deprecated in 1.10, so we'll be good to remove in 1.11.

Reply:
sure, no problem, does it mean that we actually want to have a scale endpoint for jobs ?

https://github.com/kubernetes/kubernetes/pull/58468

Reply:
> sure, no problem, does it mean that we actually want to have a scale endpoint for jobs ?
> (#58468)

No.  We'll remove the special case here and you'll no longer be able to scale them from kubectl since it wasn't scaling in the same sense.